### PR TITLE
Run Analysis tool.

### PR DESCRIPTION
**Pull request by jokva**
Adds the Run Analysis tool to the main toolbar, which gives a quick way
of running analysis on the current case, available from the main gui.

Reuses the analysis selector used in the simulation configuration
panel.

1-full-ert
2-run-analysis
3-target-dialogue
4-analysis-selector

### CODE REVIEW

**Comment by joakim-hove** [comment]
This is good - and to the specification; but oh my - the specification has changed! We are trying to reduce "state" in general, and the *current case* is one particular obnoxious example of state. So would be good if the **Run Analysis** got an extra input field with the name of the source case - this could naturally be initialized with the current case.

File: python/python/ert_gui/tools/run_analysis/run_analysis_tool.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from ert_gui.ertwidgets import resourceIcon
+from ert_gui.ertwidgets.closabledialog import ClosableDialog
+from ert_gui.tools import Tool
+from ert_gui.tools.run_analysis import RunAnalysisPanel
+
+
+class RunAnalysisTool(Tool):
+    def __init__(self):
+        super(RunAnalysisTool, self).__init__("Run Analysis", "tools/run_analysis", resourceIcon("ide/table_import"))
+        self.__run_widget = None
```
Comment (jepebe) [inline python/python/ert_gui/tools/run_analysis/run_analysis_tool.py]: Double underscore is out of fashion in ERT. :)

**Comment by jepebe** [comment]
To follow up on Joakim's request the:

`ert_gui.ertwidgets.caseselector.CaseSelector`

Will do this. Just set `update_ert=False`

**Comment by jokva** [comment]
Adress'd.

**Pull request merged**